components/public-api-server/pkg/apiv1/team.go

```diff
@@ -278,9 +278,14 @@ func (s *TeamService) toTeamAPIResponse(ctx context.Context, conn protocol.APIIn
 	}
 
 	invite, err := conn.GetGenericInvite(ctx, team.ID)
+
 	if err != nil {
-		logger.WithError(err).Error("Failed to get generic invite.")
-		return nil, proxy.ConvertError(err)
+		connectError := proxy.ConvertError(err)
+		// code not found is expected if the organization is SSO-enabled
+		if connectError != nil && connectError.Code() != connect.CodeNotFound {
+			logger.WithError(err).Error("Failed to get generic invite")
+			return nil, connectError
+		}
```

Comment:
This is slightly round-about way of getting the code. Given the error comes from the WS protocol, I'd have expected we get the code from there, rather than post-conversion to connect errors.

Here, I'd extract a helper out of the `ConvertError` function which checks if the error is a WS 404 error, and use that here, but also update the ConvertError one to use it.

Reply:
In any case, would you be able to add a test for the case where the Org is SSO enabled and confirm it doesn't have the invite populated?

Reply:
> Given the error comes from the WS protocol, I'd have expected we get the code from there, rather than post-conversion to connect errors.

I chose to do it like that, because we don't have any deps to `jsonrpc2` in this file but do use `connect` everywhere and also do the conversion.

> In any case, would you be able to add a test for the case where the Org is SSO enabled and confirm it doesn't have the invite populated?

I have added a test case, though I question they are worth a lot given that the team service mostly delegates to the jsonrpc protocol which is entirely mocked out. I.e. there is very little logic being tested in. `team_test-go`.